docker/env.sh

```diff
@@ -2,6 +2,16 @@
 
 # This file is used to set environment variables for the build.sh and run.sh scripts.
 
+# docker is the default container engine, but should work with "podman" (rootless) too!
+if [[ -z "${CONTAINER_ENGINE}" ]]; then
+    CONTAINER_ENGINE="docker"
+fi
```

Comment:
why not using `CONTAINER_ENGINE=${CONTAINER_ENGINE:-docker}`?

Reply:
Because I'm a dummy.  I can fix that!

Reply:
Nah - didn't mean it like that, but since I am always interested in learning new tricks I thought there could be a reason for the if statement 😅

Reply:
No, no I really am a dummy.  Standby for a new commit coming soon!